wrappers/src/dictionary_cs.cpp

```diff
@@ -155,11 +155,16 @@ extern "C" {
         delete dictionary;
     }
 
```

Comment:
I'm starting to believe that these colleciton-methods could be merged into something like

`realm_collection_add_notification_callback(object_store::Collection* collection, void* managed_collection, NativeException::Marshallable& ex)`